### PR TITLE
[wasm] add '--use_extensions' for wasm build by default

### PR DESCRIPTION
**Pull request by fs-eire**
**Description**: add '--use_extensions' for wasm build by default

### CODE REVIEW

**Comment by snnn** [comment]
Is it needed for this release? If not, I suggest postponing it. ORT-extension hasn't done any binary release yet. We need to examine what third-party libraries will be added to the binaries, and update https://github.com/microsoft/onnxruntime/blob/main/cgmanifests/cgmanifest.json , and ThirdPartyNotice.txt, a lot of things to do.

File: tools/ci_build/github/azure-pipelines/templates/win-wasm-ci.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
   variables:
     EnvSetupScript: setup_env.bat
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/templates/win-wasm-ci.yml:30]: BTW, this bat is only for C# tests. Normal C/C++ or python build do not need this.

**Comment by skottmckay** [comment]
Should this be a separate build/package so a user has a choice of a base ORT package or a much larger one with extensions? Or will we see how big the binary size is and decide if it's worth having an ORT and ORT+extensions package?

**Comment by fs-eire** [comment]
replaced by #14239

**Pull request closed without merging**